### ISSUE TITLE
Create Schedule should enable start and stop values by using flow tags

### ISSUE THREAD

**Issue by bwa** [ricott/homey-no.easee#55]
I have a case when I have a flow that has a dynamic start and stop time. It's not possible to execute action Create Schedule using #start-time and #stop-time from the flow.

**Comment by n808** [comment]
Yes, this feature is needed. The Create Schedule card is rather pointless without support for dynamic variables. If a Pull request was added with this implemented, perhaps the feature would be added.

**Comment by Bjorn-Willy-Arntzen** [comment]
For now I have created a HomeyScript to transform the flow variable to a string, it works, but it's not pretty.
